SteamList/GamesViewController.swift

```diff
@@ -10,13 +10,25 @@ import UIKit
 class GamesViewController: UIViewController {
 
     @IBOutlet weak var tableView: UITableView!
-    private let numberOfGames = 100
+    
+    let cellId = "GameTableViewCell"
```

Comment:
I would suggest to make an extension of UITableViewCell to hold its id. Or at least hold it statically

Reply:
done